scipy/stats/tests/test_stats.py

```diff
@@ -292,6 +292,8 @@ class TestFisherExact(object):
     def test_basic(self):
         fisher_exact = stats.fisher_exact
 
+        res = fisher_exact([[5829225, 5692693], [5760959, 5760959]])[1]
+        assert_approx_equal(res, 3.06311e-178, significant=4)
```

Comment:
I asked R for the value and got 
```
> fisher.test(matrix(c(5829225, 5692693, 5760959, 5760959), nrow=2, ncol=2))[1]
$p.value
[1] 6.126213e-178
```

Which is not significant = 4

Reply:
Looks like the value `3.06311e-178` is wrong.  I did an independent calculation (a slow, brute force method using the Gnu GMP library) and got a p-value of `6.12621271262e-178`.  That matches the R result up to the number of digits that I recorded:

```
> options(digits=12)
> result = fisher.test(matrix(c(5829225, 5692693, 5760959, 5760959), nrow=2, ncol=2))
> result$p.value
[1] 6.12621271262e-178
```

Reply:
Hold on, though.  I also calculated the equivalent of `alternative='greater'` and that value is `3.06310635631e-178`, which also agrees with R:
```
> result = fisher.test(matrix(c(5829225, 5692693, 5760959, 5760959), nrow=2, ncol=2), alternative='greater')
> result$p.value
[1] 3.06310635631e-178
```
Just noting that fact for future reviewers.  I haven't looked into why the above test ended up with what appears to be the result for the case `alternative='greater'`.